framework/generated/khronos_generators/vulkan_generators/base_generator.py

```diff
@@ -844,6 +842,10 @@ def getBaseOutputStructureName(self):
         """Method override"""
         return 'VkBaseOutStructure'
 
+    def getStructTypeEnumName(self):
```

Comment:
Why isn't this (and related in (or referenced from) the ApiPrefix object?

Reply:
It does in the future.